dell-r730/workflows/dell-perccli-create-megaraid-graph.json

```diff
@@ -1,12 +1,33 @@
+root@kcps-rackhd01:/home/wangzz/on-skupack/dell-r730/workflows# cat dell-perccli-create-megaraid-graph.json
 {
     "friendlyName": "Create RAID via perccli",
     "injectableName": "Graph.Raid.Create.Perccli",
     "options": {
         "bootstrap-ubuntu": {
-            "overlayfsUri": "{{ api.server }}/common/dell.raid.overlay.cpio.gz"
+            "overlayfsUri": "{{ api.server }}/common/dell.raid.overlay.cpio.gz",
+            "triggerGroup": "bootstrap"
+        },
+        "finish-bootstrap-trigger":{
+            "triggerGroup": "bootstrap"
         },
         "create-raid": {
-            "path": "/opt/MegaRAID/perccli/perccli64"
+            "path": "/opt/MegaRAID/perccli/perccli64",
```

Comment:
options will not be run if createDefault/deleteAll is true
leave the options could help people know what options could be used in for this task
Especially people who don't know MUSTACHD will no make sense from command